rclcpp_examples/test/test_executables_example.py.in

```diff
@@ -0,0 +1,61 @@
+from importlib.machinery import SourceFileLoader
+import os
+from launch import LaunchDescriptor
+from launch.exit_handler import ignore_exit_handler
+from launch.launcher import DefaultLauncher
+from launch_testing import InMemoryHandler
+
+
+def setup():
+    os.environ['OSPL_VERBOSITY'] = '8'  # 8 = OS_NONE
+
+
+def test_executable():
+    output_handlers = []
+
+    ld = LaunchDescriptor()
+
+    i = 0
+    for exe, output_file in zip('@RCLCPP_EXAMPLES_EXECUTABLE@'.split(';'),
+                                '@RCLCPP_EXAMPLES_EXPECTED_OUTPUT@'.split(';')):
+        name = 'test_executable_' + str(i)
+        literal_file = output_file + '.txt'
+        regex_file = output_file + '.regex'
+        py_file = output_file + '.py'
+        handler = None
+        if os.path.isfile(literal_file):
+            with open(literal_file, 'rb') as f:
+                expected_output = f.read().splitlines()
+            handler = InMemoryHandler(name, ld, expected_output, False)
+        elif os.path.isfile(regex_file):
+            with open(regex_file, 'rb') as f:
+                expected_output = f.read().splitlines()
+            handler = InMemoryHandler(name, ld, expected_output, True)
+        elif os.path.isfile(py_file):
+            checker_module = SourceFileLoader(
+                'checker_module', py_file).load_module()
+            handler = checker_module.CheckerHandler(name, ld)
+        else:
+            assert handler, 'Cannot find appropriate handler'
```

Comment:
`handler` is always `None` here. So this could be:

    assert False, 'Cannot find appropriate handler for: %s' % output_file